ncm-metaconfig/src/test/perl/service-icinga-web.t

```diff
@@ -0,0 +1,9 @@
+use Test::More;
+use Test::Quattor::TextRender::Metaconfig;
+use XML::Parser;
+
+my $u = Test::Quattor::TextRender::Metaconfig->new(
+        service => 'icinga-web',
+        )->test();
```

Comment:
@jrha it was taken from other tests, but what would you like? 8 spaces for the service line and 4 for the test() line?